README.md

```diff
@@ -1,62 +1,58 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# COVID-19 tracking
 
-> Description the project.
+> A single page application thats displays COVID-19 information, the app shows the country name, flag active cases, number of deaths and total number recovered in a day.
 
+Get started by copying and pasting the following in your terminal:
 
-## Built With
+- 1
 
-- Major languages
-- Frameworks
-- Technologies used
+```
+git@github.com:Kayonga99/React-capstone-Covid-data.git
+```
 
-## Live Demo (if available)
+- 2
 
-[Live Demo Link](https://livedemo.com)
+```
+REACT-CAPSTONE-COVID-DATA
+```
 
+- 3
 
-## Getting Started
+```
+npm start
+```
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+```
+http://localhost:3000
+```
 
+- [click here! to see live demo]()
```

Comment:
- [ ] Add a live demo link

- [ ] Record a video presentation of the project and include the link in your readme documentation.